src/Metrics.Extensions/src/Public/Microsoft.ApplicationInsights.Metrics/MetricSeriesConfigurationForAccumulator.cs

```diff
@@ -0,0 +1,200 @@
+        {
+            _restrictToUInt32Values = restrictToUInt32Values;
+
+            _hashCode = Util.CombineHashCodes(_restrictToUInt32Values.GetHashCode());
+        }
+
+        /// <summary />
+        public bool RequiresPersistentAggregation
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            get { return true; }
+        }
+
+        /// <summary />
+        public bool RestrictToUInt32Values
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            get { return _restrictToUInt32Values; }
+        }
+
+        /// <summary />
+        /// <param name="dataSeries"></param>
+        /// <param name="aggregationCycleKind"></param>
+        /// <returns></returns>
+        public IMetricSeriesAggregator CreateNewAggregator(MetricSeries dataSeries, MetricAggregationCycleKind aggregationCycleKind)
+        {
+            IMetricSeriesAggregator aggregator = new AccumulatorAggregator(this, dataSeries, aggregationCycleKind);
+            return aggregator;
+        }
+
+        /// <summary />
+        /// <param name="obj"></param>
+        /// <returns></returns>
+        public override bool Equals(object obj)
+        {
+            if (obj != null)
+            {
+                var otherConfig = obj as MetricSeriesConfigurationForAccumulator;
+                if (otherConfig != null)
+                {
+                    return Equals(otherConfig);
+                }
+            }
+
+            return false;
+        }
+
+        /// <summary />
+        /// <param name="other"></param>
+        /// <returns></returns>
+        public bool Equals(IMetricSeriesConfiguration other)
+        {
+            return Equals((object) other);
+        }
+
+        /// <summary />
+        /// <param name="other"></param>
+        /// <returns></returns>
+        public bool Equals(MetricSeriesConfigurationForAccumulator other)
+        {
+            if (other == null)
+            {
+                return false;
+            }
+
+            if (Object.ReferenceEquals(this, other))
+            {
+                return true;
+            }
+
+            return (this.RestrictToUInt32Values == other.RestrictToUInt32Values);
+        }
+
+        /// <summary />
+        /// <returns></returns>
+        public override int GetHashCode()
+        {
+            return _hashCode;
+        }
+
+        /// <summary>
+        /// Groups constants used by metric aggregates produced by aggregators that are configured by metric configurations represented through
+        /// instances of <see cref="MetricSeriesConfigurationForAccumulator"/>. This class cannot be instantiated. To access the constants, use the 
+        /// extension method <c>MetricConfigurations.Common.Accumulator().Constants()</c> or <see cref="MetricSeriesConfigurationForAccumulator.Constants"/>.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
why not browsable?

Reply:
The class does not need to pollute intellisense in the metrics namespace. the extensions methods its defines are still browsable after the dot of the this parameter.